src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2386,7 +2403,7 @@ export class StudyViewPageStore
         this.changeChartVisibility(uniqueKey, true);
     }
     isXVsYChartVisible(attrIdA: string, attrIdB: string) {
-        const allCharts = this._xVsYChartMap.entries();
+        const allCharts = this._xVsYScatterChartMap.entries();
```

Comment:
maybe change the name of this from allCharts to just "charts" .   allcharts jumped out at me inside a method that deals only with xvsycharts